### PR TITLE
Allow sendmail to anyone

### PR DESCRIPTION
**Pull request by nickumia-reisys**
Related to https://github.com/GSA/data.gov/issues/3810

### CODE REVIEW

**Comment by nickumia-reisys** [comment]
Blocked by https://docs.aws.amazon.com/ses/latest/dg/request-production-access.html?icmpid=docs_ses_console

**Comment by nickumia-reisys** [comment]
Manual test based on https://www.databulle.com/blog/code/python-emails-amazon-ses.html

**Review by jbrown-xentity — APPROVED** [review]

**Review by jbrown-xentity — APPROVED** [review]

**Comment by btylerburton** [comment]
I missed the boat. Looks good, though. 👍🏼

File: terraform/bind/main.tf [hunk 1]
```diff
@@ -27,9 +27,9 @@ resource "aws_iam_user_policy" "user_policy" {
         "ses:SendEmail",
         "ses:SendRawEmail"
       ],
-      "Resource": "${var.domain_arn}"
+      "Resource": "*"
```
Comment (mogul) [inline terraform/bind/main.tf:30]: I don't think this is correct... See https://github.com/GSA/data.gov/issues/3810#issuecomment-1117997318

Comment (nickumia-reisys) [reply terraform/bind/main.tf:30]: It's reverted in #16

**Pull request merged**